packages/core/src/query/flux.ts

```diff
@@ -0,0 +1,245 @@
+const FLUX_VALUE = Symbol.for('FLUX_VALUE')
```

Comment:
imho, the symbol should not be stored in the global registry, but exported as a module member. afaik there is no conventional naming schema to avoid conflicts. i only ever seen exported symbols until now

Reply:
it is this way now